vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -982,7 +986,8 @@ testWasmRuntimesInstalled () {
   # v0.15.1 does not have a version encoded in the binary name
   binary_path_pattern="${wasm_runtimes_path}/containerd-shim-spin-v2"
     if [ ! -f $binary_path_pattern ]; then
-      err "$test: Spin Wasm Runtime binary does not exist at $binary_path_pattern"
+      output=$(ls -la $binary_path_pattern)
+      err "$test: Spin Wasm Runtime binary does not exist at $binary_path_pattern /// $output"
       return 1
     else
       echo "$test: Spin Wasm Runtime binary exists at $binary_path_pattern"
```

Comment:
suggest: well actually we have a special case for SpinKube already, so no need to modify the test above to include spinkube. SpinKube always starts from version `v0.15.1`